types/events.go

```diff
@@ -102,13 +104,14 @@ type MemoryEntry struct {
 }
 
 type Memory struct {
-	Cache     uint64            `json:"cache,omitempty"`
-	Usage     MemoryEntry       `json:"usage,omitempty"`
-	Swap      MemoryEntry       `json:"swap,omitempty"`
-	Kernel    MemoryEntry       `json:"kernel,omitempty"`
-	KernelTCP MemoryEntry       `json:"kernelTCP,omitempty"`
-	Raw       map[string]uint64 `json:"raw,omitempty"`
-	PSI       *PSIStats         `json:"psi,omitempty"`
+	Cache            uint64            `json:"cache,omitempty"`
+	Usage            MemoryEntry       `json:"usage,omitempty"`
+	Swap             MemoryEntry       `json:"swap,omitempty"`
+	Kernel           MemoryEntry       `json:"kernel,omitempty"`
+	KernelTCP        MemoryEntry       `json:"kernelTCP,omitempty"`
+	Raw              map[string]uint64 `json:"raw,omitempty"`
+	PSI              *PSIStats         `json:"psi,omitempty"`
+	MemoryEventCount MemoryEventCount  `json:"memory_event_count"`
```

Comment:
Why not use `EventCount`, I think the prefix `Memory` is redundant, because it's in the `Memory` struct.